Cargo.toml

```diff
@@ -13,8 +13,9 @@ edition = "2018"
 
 [features]
 default = [ "std" ]
-std = [ "unicode-normalization", "serde/std" ]
+std = [  "serde/std", "alloc" ]
 rand = [ "crate_rand", "rand_core" ]
+alloc = [ "unicode-normalization" ]
```

Comment:
If we move this here we likely want to change the dependency to
```rust
unicode-normalization = { version = "=0.1.22", default-features=false, optional = true }
```

Reply:
6f967f1